### PR TITLE
widgettoggler 1.2.0 (new cask)

### PR DESCRIPTION
**Pull request by bobbymannino**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Comment by bevanjkay** [comment]
Thank you for submitting this @bobbymannino 
Please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

My other concern is that this hasn't been updated for over a year, and a recent [readme update](https://github.com/sieren/WidgetToggler/commit/55766995e70be5480ce9c6b6479d038f15b94e16) suggests that the primary use case is no longer relevant.

File: Casks/w/widgettoggler.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask "widgettoggler" do
+  version "1.2.0"
+  sha256 "4a37bd84eb391c86e80a05b3b875dae6286418f2223970dbeff03abe310b5af9"
+
+  url "https://github.com/sieren/WidgetToggler/releases/download/#{version}/WidgetToggler_#{version.split(".")[0...2].join(".")}.zip"
+  name "widgettoggler"
```
Comment (khipp) [inline Casks/w/widgettoggler.rb:6]:
```suggestion
  name "WidgetToggler"
```

File: Casks/w/widgettoggler.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cask "widgettoggler" do
+  version "1.2.0"
+  sha256 "4a37bd84eb391c86e80a05b3b875dae6286418f2223970dbeff03abe310b5af9"
+
+  url "https://github.com/sieren/WidgetToggler/releases/download/#{version}/WidgetToggler_#{version.split(".")[0...2].join(".")}.zip"
```
Comment (khipp) [inline Casks/w/widgettoggler.rb:5]:
```suggestion
  url "https://github.com/sieren/WidgetToggler/releases/download/#{version}/WidgetToggler_#{version.major_minor}.zip"
```

Comment (bobbymannino) [reply Casks/w/widgettoggler.rb]: added In the fix commit

File: Casks/w/widgettoggler.rb (hunk 1 above)
Comment (bobbymannino) [reply Casks/w/widgettoggler.rb]: added In the fix commit

**Comment by bobbymannino** [comment]
I have made the changes suggested, and added a zap. the repo itself suggests that no updated will be coming but the app works as is so its up to you wether to keep this or not.

**Review by p-linnane — APPROVED** [review]
Thanks!

**Pull request merged**